packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -58,38 +56,46 @@ export interface ITree extends IChannel {
 	 * Additionally, once out of schema content adapters are properly supported (with lazy document updates),
 	 * this initialization could become just another out of schema content adapter and this initialization is no longer a special case.
 	 */
-	schematize<TRoot extends ImplicitFieldSchema>(
+	viewWith<TRoot extends ImplicitFieldSchema>(
```

Comment:
We should make this method async to avoid a future breaking change.

Eventually, we will want to expand TreeConfiguration to specify a branch and which subset of the tree data we want to be included in our view.

When that happens, this method will need to be async.  (Otherwise, we would need to have already downloaded the full dataset for all branches in order to synchronously return the view.)

Reply:
Works for me. I did confirm that this seems like it shouldn't really be a burden for prod apps, as they'll typically be in an async context (`IRuntimeFactory` using lower level APIs, hasInitialized / initializingFirstTime if using aqueduct) already.

Updated.